### PR TITLE
ENH: Start and end of a segment are more informative than midpoint only

### PR DESCRIPTION
**Pull request by dzenanz**
When experimenting with different fixed depths and different padding amounts, it is more useful to see the actual pixel pairs which participated in computing the attenuation than just their midpoint. Especially when overlaying this with original segmentation.

### CODE REVIEW

**Comment by tbirdso** [comment]
@dzenanz Could you briefly describe the motivation for this change? Updated behavior looks functional.

**Comment by dzenanz** [comment]
I just entered PR description.

File: include/itkAttenuationImageFilter.hxx [hunk 1]
```diff
@@ -277,13 +277,14 @@ AttenuationImageFilter<TInputImage, TOutputImage, TMaskImage>::ThreadedGenerateD
             }
             float estimatedAttenuation = ComputeAttenuation(target, start);
 
-            // Compute mid-point index
-            target[m_Direction] = (start[m_Direction] + target[m_Direction]) / 2;
+            // Record this attenuation for both pixels of the pair
+            output->SetPixel(start, estimatedAttenuation);
             output->SetPixel(target, estimatedAttenuation);
```
Comment (tbirdso) [inline include/itkAttenuationImageFilter.hxx:282]: Does this result in the metric image pixel value at `target` being overwritten multiple times as pairs are computed?

Comment (dzenanz) [reply include/itkAttenuationImageFilter.hxx:282]: This is the pairs case, where only one pair per segment is done. Lines 228-268 are for per pixel estimation.

**Review by tbirdso — APPROVED** [review]
LGTM

**Pull request merged**